src/PrettyPrinting.jl

```diff
@@ -1591,23 +1591,81 @@ function _write_line(io::IOCustom, str::AbstractString)
     spaceleft = c - ind - io.printed
   end
   #@show spaceleft
-  firstlen = min(spaceleft, length(str))
-  firststr = str[1:firstlen]
+  # The following code deals with line wrapping of Unicode text, including
```

Comment:
Can't we just do this at the start and then remove the last lowercase bits afterwards?
```suggestion
  if io.lowercasefirst
    str = lowercasefirst(str)
    io.lowercasefirst = false
  end 
  # The following code deals with line wrapping of Unicode text, including
```